KBoard/KBoard/View/DictionaryViewUIComponent/CategoryListView.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  CategoryListView.swift
+//  KBoard
+//
+//  Created by ParkJunHyuk on 2022/07/27.
+//
+
+import UIKit
+
+class CategoryListView: UIView {
+    
+    let categoryTitle = ["All","Happy","Sad","Angry","Good","All","Happy","Sad","Angry","Good"]
```

Comment:
```suggestion
    let categoryTitle = ["All", "Happy", "Sad"...]
```
목업데이터지만 굳~~~이 말해보자면 콤마 뒤에 한칸 띄어 주는 것이 가독성을 높혀줍니다!!

Reply:
넵! 가독성을 높히기 위해서 띄어쓰기 추가했습니다!